aiohttp/web_exceptions.py

```diff
@@ -122,11 +123,80 @@ def __init__(self, *,
         self._reason = reason
         self._text = text
         self._headers = real_headers
+        self._cookies: 'SimpleCookie[str]' = SimpleCookie()
         self.args = ()
 
     def __bool__(self) -> bool:
         return True
 
+    @property
+    def cookies(self) -> 'SimpleCookie[str]':
```

Comment:
Please not copy this block from `web_response.py` but reuse.
Maybe extraction the common code into some mixin class in `helpers.py` can handle it pretty well.

Reply:
Sounds right for me, code duplication it's bad way.

Reply:
OK, as we can place this code in `helpers.py`? I think new class `CookiesMixin` will be good solution.

Reply:
sounds good